src/pudl/package_data/eia860/column_maps/boiler_info.csv

```diff
@@ -0,0 +1,39 @@
+year_index,2009,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020
+utility_id_eia,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id
+utility_name_eia,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name
+plant_id_eia,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code
+plant_name_eia,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name
+state,state,state,state,state,state,state,state,state,state,state,state,state
+boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id
+hrsg,name_of_water_source,name_of_water_source,,,,,,hrsg,hrsg,hrsg,hrsg,hrsg
```

Comment:
This is a funny one. Are `hrsg` and `name_of_water_source` really referring to the same value?

Reply:
HRSG means heat recovery steam generator so not even close. That column showed up in 2016 so I've removed the mapping for previous years.

Reply:
We should keep the `name_of_water_source` in there somewhere if it exists.

There's [a separate little adjunct table to the EIA-923](https://www.eia.gov/electricity/data/water/) that describes the the cooling systems and water consumption for thermal plants at the individual boiler & generator level that I think could be valuable if we ever want to analyze the vulnerability of fossil generation to increasing temperatures and water scarcity.

Reply:
I do keep the `name_of_water_source` in `boiler_info` when it exists, which is only in 2009 and 2010. For those years, `name_of_water_source` is on every tab of the EnviroEquip workbook. For `cooling_equipment` those years have both `name_of_water_source` and `cooling_water_source` which are almost always the same. I had been dropping `name_of_water_source` on that tab because mapping it to `cooling_water_source` caused an error because it resulted in duplicate column names and it seemed unnecessary. But after looking more closely, it is more complete so I'll keep both.

In latter years water source is only identified for cooling where it is called `cooling_water_source`. I have followed this sort of approach in naming water sources that show up on specific equipment tabs so for `fgp_equipment` `name_of_water_source` becomes `fgp_water_source`. My logic is that if we want to combine these tables into a single environmental equipment table, we'll want to know what water source the column refers to.

Reply:
I do keep the `name_of_water_source` in `boiler_info` when it exists, which is only in EviroEquip in 2009 and 2010. After that, it migrates to Plant so I will conform the names to `water_source` as used there.